monitoring/data_hub_pipeline_health_check.py

```diff
@@ -0,0 +1,132 @@
+import os
+import logging
+import pandas as pd
+import requests
+from io import StringIO, BytesIO
+import boto3
+
+LOGGER = logging.getLogger(__name__)
+
+
+def read_big_query(query: str, **kwargs):
+    return pd.read_gbq(
+        query,
+        dialect='standard',
+        progress_bar_type=None,
+        **kwargs
+    )
+
+
+def get_query(
+        project: str,
+        dataset: str,
+        table: str):
+    return 'SELECT * FROM {project}.{dataset}.{table}'.format(
+        project=project,
+        dataset=dataset,
+        table=table
+    )
+
+
+def read_dataframe_from_s3_bucket(
+    bucket_name: str,
+    object_name: str
+) -> pd.DataFrame:
+    s3_object = boto3.client('s3').get_object(Bucket=bucket_name, Key=object_name)
+    return pd.read_csv(BytesIO(s3_object['Body'].read()))
```

Comment:
whenever there is a `read` from a stream, I am thinking it probably needs a context manager to ensure the stream is closed afterwards.

There are BTW some S3 related utilites in https://github.com/elifesciences/data-hub-core-airflow-dags/blob/develop/data_pipeline/utils/data_store/s3_data_service.py
Could also add a `read_bytes_from_s3` or similar.
Pandas can also read from s3 if you give it a URL, e.g. `s3://bucket/object` (it will need the related `fsspec` dependencies)